assets/sass/components/_alerts.scss

```diff
@@ -0,0 +1,19 @@
+
+.error-summary-outer {
```

Comment:
Could we try wrapping these styles inside a container with a single class? Take a look at #631 for the pattern. 

It's commonly used in methodologies like [BEM (Block, element modifier)](http://getbem.com/introduction/), and SASS makes it easy with nesting. If every new "component" has this style of container, there's much less change of styles overwriting each other by accident.

Reply:
These were very good points, your input was very welcomed here! I implemented them now. I was trying to think of a way to give descriptive name for the different classes. Let me know, if you think these are ok, or whether you had something else in mind.

Reply:
This looks good 👍